scripts/west-commands.yml

```diff
@@ -1,5 +1,10 @@
 # Keep the help strings in sync with the values in the .py files!
 west-commands:
+  - file: scripts/west_commands/boards.py
+    commands:
+      - name: boards
```

Comment:
I wonder if this should be instead `board`.
Then we could have `west board -a/--all` to list all boards, but we could also do things like:
`west board reel_board` to display info about a single board.

Thoughts?

Reply:
On the other side, `west boards` is extremely intuitive, so that is a big plus for keeping it as is, for a command that will likely be invoked many times.

Reply:
If the predominant use is to specify a single board, that would indicate using the singular ``board`` command name.